src/v/redpanda/admin_server.cc

```diff
@@ -3446,6 +3446,21 @@ void admin_server::register_debug_routes() {
               [] { return ss::json::json_return_type(ss::json::json_void()); });
       });
 
+    register_route<user>(
+      ss::httpd::debug_json::refresh_disk_health_info,
+      [this](std::unique_ptr<ss::httpd::request>) {
+          vlog(logger.info, "Request to refresh disk health info");
+          return _metadata_cache
+            .invoke_on_all([](auto& mc) { return mc.refresh_health_monitor(); })
```

Comment:
is this still relevant? it looks like this will result in a call per core to refresh which will then be redirected to the health monitor core. is there any per-core state to clear?

Reply:
From what I see, the only state the  frontend has is `_cluster_disk_health` but the frontend will process a refresh on the `refresher_shard` and then propogate the updates to all other shards.

So, I think the change I should do here is `_metadata_cache.local().refresh_health_monitor()` and then let the frontend dispatch the refresh request to the `refresher_shard`.